android-sdk/src/main/java/org/sagebionetworks/bridge/android/util/ScheduledActivityUtil.java

```diff
@@ -0,0 +1,288 @@
+/*
+ *    Copyright 2018 Sage Bionetworks
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package org.sagebionetworks.bridge.android.util;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.base.Function;
+import com.google.common.base.Verify;
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Multimaps;
+
+import org.sagebionetworks.bridge.rest.model.ScheduledActivity;
+import org.sagebionetworks.bridge.rest.model.TaskReference;
+
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+
+/**
+ * Created by liujoshua on 2/7/2018.
+ */
+
+@SuppressWarnings("Convert2Lambda")
```

Comment:
Simply curious, why not just convert to lambda?

Reply:
I wanted to use @NonNull and @Nullable annotations and have Studio stop suggesting I convert to a lambda.